### PR TITLE
[ADP-3409] Boostrap mainnet node via mithril

### PR DESCRIPTION
**Pull request by paolino**
- [x] Docker based mithril bootup buildkite step, https://buildkite.com/cardano-foundation/cardano-wallet/builds/6843#01915ab0-486b-43c2-a963-7f5c215b726e/220-237
- [x] Nix based mithril bootup buildkite step , https://buildkite.com/cardano-foundation/cardano-wallet/builds/6843#01915ab0-4828-4fe2-82f2-03ffd856303b/6-23

ADP-3409

### CODE REVIEW

File: run/mainnet/nix/.env [hunk 1]
```diff
@@ -1 +1,3 @@
 NETWORK="mainnet"
+AGGREGATOR_ENDPOINT=https://aggregator.release-mainnet.api.mithril.network/aggregator
```
Comment (paweljakubas) [inline run/mainnet/nix/.env:2]: this is the same env variables like for docker - we don't want to reuse them?

**Comment by paweljakubas** [comment]
You are proving here in this PR  mithril can be switched on and we will have proper syncing, correct? I bet any benchmarking will be topic of next PRs, right?

**Review by paweljakubas — APPROVED** [review]
Nice work. We should be good modulo rebase conflict resolution. You could also consider .env reuse

File: run/mainnet/nix/.env (hunk 1 above)
Comment (paolino) [reply run/mainnet/nix/.env:2]: reusing the .env is possible with symlinks, but I'm not sold on the design of this `run` directory yet. I will happily reconsider this in future PRs. Good point anyway.

**Comment by paolino** [comment]
The benchmark is the buildkite step runtime. The node (mainnet) is synced in less than 1 hour, which means an empty wallet will be synced in 2 hours.

**Pull request merged**